File: Python/avulsos/exercicio8.py
sal_hora = float(input("Digite quanto você ganha por hora: "))
horas = int(input("Quantas horas você trabalha no mês?: "))
while sal_hora < 1 or horas < 1:
  print("Digite valores válidos!")
  sal_hora = float(input("Digite quanto você ganha por hora: "))
  horas = int(input("Quantas horas você trabalha no mês?: "))

salario = sal_hora * horas
print(f"Seu salário é de R${salario:.2f}")

File: C/exercicio8.c
#include <stdio.h>

int main(void)
{
  float sal_hora;
  int horas;
  printf("Digite o número o seu ganho por hora e o número de horas trabalhadas no mês: ");
  scanf("%f %i", &sal_hora, &horas);
  while (sal_hora < 1 || horas < 1)
    {
      printf("Digite valores válidos!\n");
      printf("Digite o número o seu ganho por hora e o número de horas trabalhadas no mês: ");
      scanf("%f %i", &sal_hora, &horas);
    }
  float salario = sal_hora * horas;
  printf("Seu salário é de R$%.2f\n", salario);
  return 0;
}